core/plugins/org.fusesource.ide.camel.model.service.core/src/org/fusesource/ide/camel/model/service/core/catalog/cache/CamelCatalogCacheManager.java

```diff
@@ -27,6 +32,7 @@
 	private static Map<CamelCatalogCoordinates, CamelModel> camelModelCache = new HashMap<>();
 
 	private static final CamelCatalogCacheManager instance = new CamelCatalogCacheManager();
+	private static CamelModel lastRetrievedCamelCatalog = null;
```

Comment:
do we really need to initialize it with null?

Reply:
there is a check to know if there is a value or not so it would make an error with "Not initialized" attribute.

The other way is to initialize it with CamelModel for default Camel Model but it will require to compute it even if we don't use it

Reply:
Which check are you talking about? Java does set not initialized variables like this to null automatically if not set explicitely.

Reply:
my bad I mixed with local variables. Reminder for later me:
- class attributes are initialized by default value (null for object which are not primitives)
- local variables must be initialized

thanks!